From fc2485d62d0a838c48cadc0cd0bfc03ea49407ef Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@intel.com>
Date: Thu, 25 Nov 2010 01:23:51 -0600
Subject: [PATCH 05/15] eg20t: Fix pch_dma compile error

DMA_ENGINE needs to be selected in order for drivers/dma to be
added to the compile.

Signed-off-by: Tom Zanussi <tom.zanussi@intel.com>
---
 drivers/dma/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/dma/Kconfig b/drivers/dma/Kconfig
index 3957084..4fccb68 100644
--- a/drivers/dma/Kconfig
+++ b/drivers/dma/Kconfig
@@ -39,6 +39,7 @@ config ASYNC_TX_DISABLE_CHANNEL_SWITCH
 config PCH_UART_DMA
 	tristate "PCH DMA Controller"
 	depends on PCI && SERIAL_8250_PCH_DMA
+	select DMA_ENGINE
 	default y
 	help
 	  This value must equal to SERIAL_8250_PCH. This config PCH_UART_DMA is
-- 
1.7.0.4

